### ISSUE TITLE
Enhancement suggestion - stats panel

### ISSUE THREAD

**Issue by Disorganise** [EDDiscovery/EDDiscovery#2031]
In the statistics panel/travel tab, the columns are broken into 24hrs, week, month, last dock. trip and all.

I think it might be useful to have 'session', meaning this session.  Sometimes on expeditions, you need to keep up a pace, the 24hrs column doesn't tell the whole story as jumps fall off the tail depending how much time was spent in system.  Having a session column would allow easy reference to when 1kly, or whatever the target is, has been completed.

*Also*, is it possible to add a couple of rows?:
Elapsed Game Time [Time count between starting and ending game sessions.  I guess this might be tricky to calculate when there have been game crashes, but maybe the first and last timestamp of each journal could be used?]
Average Time per system [Elapsed Game Time divided by number of jumps]

I'd like to see if I'm spending more or less time on average in systems now, and it'll be interesting once the new content is released.  Also interested to know how much time this game really absorbs!

**Comment by spadino** [comment]
I worked some time ago on an overhaul of the stats panel, but do not committed that, as there are more urgent needs. Now that we have the splits container, which is far more usable than grid, I'll work again on that...
Beside the features you suggested, I worked on various statistical graphs, which can be opened also as standalone panels. 

One of the things I would like to address before, however, is the tab implementation. I think that no panel should be hardcoded in subcontainers like tabs, yet have a tab container like splits and grids, which can host any number of panel inside, and which can be hosted inside a container, it can be extremely helpfull.

@robbyxp is working on a preset feature, which enable us to have presets for pages. That way, we can have a "statistics" page preset, which present all statiscal panels and modules pre-arranged. This can be extended to sub-containers, so to let a tab subcontainer load something similar to the current stats panel implementation.
